src/main/java/br/com/moip/resource/Invoice.java

```diff
@@ -1,12 +1,14 @@
 package br.com.moip.resource;
 
 import java.util.Date;
+import java.util.List;
 
 public class Invoice {
-    private String id, description;
-    private int invoiceAmount;
-    private Date createdAt, updatedAt;
 
+    private String id;
```

Comment:
Here comes the `external_id`? If yes, what do you think in change the name to `external_id`?